Library/Homebrew/brew_irbrc

```diff
@@ -4,6 +4,6 @@
 
 require 'irb/completion'
 
-IRB.conf[:SAVE_HISTORY] = 100
 IRB.conf[:HISTORY_FILE] = "#{Dir.home}/.brew_irb_history"
 IRB.conf[:IRB_NAME] = "brew"
+H = Homebrew
```

Comment:
Do we want to set this in `dev-cmd/irb` instead so that it works with pry too? Or would that pollute the environment too much? I can see it maybe being a problem in tests were all files get loaded at least once.